response_operations_ui/templates/collection-exercise.html

```diff
@@ -0,0 +1,548 @@
+{% extends "layouts/base.html" %}
+{% block page_title %}{{survey.surveyRef}} {{survey.shortName}} {{ce.exerciseRef}} | Surveys | Survey Data Collection{% endblock %}
```

Comment:
Is there a need for the `collection_exercises` directory in `templates`now, looks like you've merged them all into one file?
I wouldn't think of merging all of this into one file for this PR, doesn't look like it's be used anywhere?

Reply:
Addressed in next commit

Reply:
It seems to be used for exporting, so I've put it back.